tests/base/develop_utils.py

```diff
@@ -72,8 +72,7 @@ def assert_ok_model_acc(trainer, key='test_acc', thr=0.5):
     assert acc > thr, f"Model failed to get expected {thr} accuracy. {key} = {acc}"
 
 
-def reset_seed():
-    seed = RANDOM_SEEDS.pop()
+def reset_seed(seed=0):
```

Comment:
since reset_seed did not take the seed param before, everything now will default to seed=0, is that the expected behavior? Also, does this also set torch.manual_seed and np.random.seed, since they were removed in this PR?

Reply:
well, if we don't set a value what else can we do?
some tests may not want their random value to depend on a previous test, so reset_seed should set the same value all the time. I absolutely see no reason for the old method, essentially it did the following:

```
seed = random number
seed_everything(seed)
```

what's the point?

> also, does this also set torch.manual_seed and np.random.seed, since they were removed in this PR?

yes, seed_everyhing sets the seed in all libraries.